project/DataModule.py

```diff
@@ -34,16 +32,20 @@ class DataModule(pytorch_lightning.LightningDataModule):
 
     def __init__(
         self,
+        raw_data,
         xnat_configuration: dict = None,
         batch_size: int = 1,
         num_workers: int = 4,
         visualise_training_data=True,
+        image_series_option: str = 'error',
     ):
         super().__init__()
-        self.num_workers = num_workers
-        self.batch_size = batch_size
+        self.raw_data = raw_data
         self.xnat_configuration = xnat_configuration
+        self.batch_size = batch_size
+        self.num_workers = num_workers
         self.visualise_training_data = visualise_training_data
+        self.image_series_option = image_series_option
```

Comment:
is this only relevant for image data from XNAT? if so, could that be reflected in a comment?

Reply:
Leave for now, can always delete or refactor at a later stage.

Reply:
add comments in the code to prompt future users to delete as appropriate